### PR TITLE
doc: fhir operation framework

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: docs/src/pages/guides/FHIROperationFramework.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
+---
+layout: default
+title: IBM FHIR Server - Extended Operations Framework
+date:  2020-09-29
+---
+
+# IBM FHIR Server - Extended Operations Framework
+
+The HL7 FHIR Specification describes a [light weight framework](https://www.hl7.org/fhir/r4/operations.html) of extended operations for the REST API.
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIROperationFramework.md:9]: I noticed the spec itself calls this "lightweight" but IMHO we should remove such value statements from the technical docs (similar to avoiding terms like "simply do X" or "merely do Y").

File: docs/src/pages/guides/FHIROperationFramework.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+
+This framework defines REST endpoints at the Base Server (System), scoped to specific resources (Type) and scoped to specific instances. 
+- **System** - https://my-server.com/v4/$convert - Convert from one format to another
+- **Type** - https://my-server.com/v4/Patient/$export - Retrieve all the data from the Patient compartment
+- **Instance** - https://my-server.com/v4/Patient/1-2-3-4/$export - Retrieve a specific Patient data from the Patient compartment data
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIROperationFramework.md:14]: wondering if these should have our default base path in there.  e.g. `https://my-server.com/fhir-server/api/v4`

File: docs/src/pages/guides/FHIROperationFramework.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+convert
+export
+healthcheck
+hello
+import
+```
+
+# The Specification in More Detail 
+
+Each Operation per [the specification](https://www.hl7.org/fhir/r4/operations.html#defining) as an [OperationDefinition](https://www.hl7.org/fhir/r4/operationdefinition.html) Resource.
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIROperationFramework.md:154]: as -> is ?
```suggestion
Each Operation per [the specification](https://www.hl7.org/fhir/r4/operations.html#defining) is an [OperationDefinition](https://www.hl7.org/fhir/r4/operationdefinition.html) Resource.
```

File: docs/src/pages/guides/FHIROperationFramework.md [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+
+The elements needed are: 
+- Unique Name of the Operation - `OperationDefinition.name`
+  - For instance, `"name": "import"`
+- Code used to activate the Operation - `OperationDefinition.code`
+  - For instance, `"code": "import"`
+  - The *code* is used in the CapabilitiesStatement.
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIROperationFramework.md:161]:
```suggestion
  - The *code* is used in the CapabilityStatement.
```

**Review by lmsurpre — APPROVED** [review]
LGTM

File: docs/src/pages/guides/FHIROperationFramework.md (hunk 1 above)
Comment (prb112) [reply docs/src/pages/guides/FHIROperationFramework.md:9]: sure

**Pull request merged**